include/SFML/Window/WindowBase.hpp

```diff
@@ -393,6 +394,18 @@ class SFML_WINDOW_API WindowBase : NonCopyable
     ////////////////////////////////////////////////////////////
     WindowHandle getSystemHandle() const;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Create a Vulkan rendering surface
+    ///
+    /// \param instance  Vulkan instance
+    /// \param surface   Created surface
+    /// \param allocator Allocator to use
+    ///
+    /// \return True if surface creation was successful, false otherwise
+    ///
+    ////////////////////////////////////////////////////////////
+    bool createVulkanSurface(const VkInstance& instance, VkSurfaceKHR& surface, const VkAllocationCallbacks* allocator = 0);
```

Comment:
Since `sf::WindowsBase` acts as the base for the regular `sf::Window`, does it the `createVulkanSurface()` make sense in the context of `sf::Window` or `sf::RenderWindow`?

Reply:
I don't see why not... If someone finds that they are in a special situation where they need to use Vulkan and sfml-graphics simultaneously then it shouldn't be our goal to prevent something that is theoretically possible just as a matter of principle.